helm/prometheus-rules/templates/alerting-rules/app.rules.yml

```diff
@@ -91,6 +91,22 @@ spec:
         cancel_if_outside_working_hours: "true"
         severity: page
         topic: releng
+    - alert: ClusterCapiAppFailed
+      annotations:
+        description: '{{`Cluster CAPI App {{ if $labels.exported_namespace }}{{ $labels.exported_namespace }}{{ else }}{{ $labels.namespace }}{{ end }}/{{ $labels.name }}, version {{ $labels.version }} is {{if $labels.status }} in {{ $labels.status }} state. {{else}} not installed. {{end}}`}}'
+        opsrecipe: app-failed/
+      # we have this Frankenstein for namespace checks because go (and prometheus) don't support negative lookaheads in regex
+      expr: label_replace(app_operator_app_info{app=~"cluster-.+", status!~"(?i:(deployed|cordoned|not-installed))", catalog="cluster", team!~"^$|noteam"}, "cluster_id", "$1", "namespace", {{ include "namespaceNotGiantswarm" . }}) == 1
+      for: 30m
+      labels:
+        area: managedservices
```

Comment:
isn't this kaas? this is an app managed by kaas to go alongside a cluster, so I don't think it should be a managed service.
also I think we are missing the `team` label here.

Reply:
I don't know , i copied it from the other `ManagementClusterAppFailed` and `WorkloadClusterappFailed` but sure i can change it :+1:

Reply:
> also I think we are missing the team label here.

I thought the `team` label would come from the actual series ( so clippy for cluster-azure, hydra for cluster-aws and so on )

Reply:
yeah might be :+1: